docs/src/main/sphinx/functions/datetime.md

```diff
@@ -570,5 +570,14 @@ Returns the year of the [ISO week] from `x`.
 This is an alias for {func}`year_of_week`.
 :::
 
+:::{function} timezone(timestamp(p) with time zone) -> varchar 
+Returns the IANA timezone ID from `timestamp(p) with time zone`. 
```

Comment:
do we want to link to https://en.wikipedia.org/wiki/List_of_tz_database_time_zones .. but what is the timezone id there ?  I think TZ identifier .. maybe we should use the same name to be consistent .. or link to elsewhere